### PR TITLE
move `Dcs` classes to package `dependencies`

### LINKED ISSUE #1671: Move all `Dcs` classes into a separate package
We have several classes that describe the same entity `Dependencies`:
```
DcsDefault
DcsDepgraph
DcsUniquelyVersioned
DcsWithRuntime
```
It would be great to move all of them to a separate package.

### PR DESCRIPTION
**Pull request by MikhailLipanin**
closes #1671

### CODE REVIEW

**Comment by MikhailLipanin** [comment]
@volodya-lombrozo, Can you have a look pls?

File: eo-maven-plugin/src/main/java/org/eolang/maven/dependencies/DcsEachWithoutTransitive.java [hunk 1]
```diff
@@ -108,6 +109,16 @@ private static boolean eqTo(final Dependency left, final Dependency right) {
             && Objects.equals(left.getGroupId(), right.getGroupId());
     }
 
+    /**
+     * Is it our runtime dep?
+     * @param other The dep
+     * @return TRUE if it is
+     */
+    private static boolean isRuntime(final Dependency other) {
```
Comment (volodya-lombrozo) [inline eo-maven-plugin/src/main/java/org/eolang/maven/dependencies/DcsEachWithoutTransitive.java:117]: @MikhailLipanin It looks like code duplication. We have the same code in `ResolveMojo`

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]

File: eo-maven-plugin/src/main/java/org/eolang/maven/dependencies/DcsEachWithoutTransitive.java (hunk 1 above)
Comment (MikhailLipanin) [reply eo-maven-plugin/src/main/java/org/eolang/maven/dependencies/DcsEachWithoutTransitive.java:117]: @volodya-lombrozo, Yes, but in `ResolveMojo` this method need to be `public static boolean` to be used in `DcsEachWithoutTransitive`.
So, as in `DcsWithRuntime` there is `private static boolean isRuntime(final Dependency other)`, I added the same code to `DcsEachWithoutTransitive`

Comment (volodya-lombrozo) [reply eo-maven-plugin/src/main/java/org/eolang/maven/dependencies/DcsEachWithoutTransitive.java:117]: @MikhailLipanin let's make it `public`, it's much better comparing with code duplication as for me

Comment (MikhailLipanin) [reply eo-maven-plugin/src/main/java/org/eolang/maven/dependencies/DcsEachWithoutTransitive.java]: done

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**